extensions/wikia/PlaybuzzTag/PlaybuzzTagController.class.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class PlaybuzzTagController extends WikiaController {
+	const PARSER_TAG_NAME = 'playbuzz';
+	const PLAYBUZZ_SCRIPT_SRC = 'http://cdn.playbuzz.com/widget/feed.js';
+	const DATA_ITEM_ATTR = 'data-item';
+
+
+	public static function onParserFirstCallInit( Parser $parser ) {
+		$parser->setHook( self::PARSER_TAG_NAME, [ new static(), 'renderTag' ] );
+		return true;
+	}
+
+	public function renderTag( $input, array $args, Parser $parser, PPFrame $frame ) {
+		if ( !$this->validateArgs( $args ) ) {
+			return '<strong class="error">' . wfMessage( 'playbuzz-tag-could-not-render' )->escaped() . '</strong>';
+		}
+
+		$script = JSSnippets::addToStack(self::PLAYBUZZ_SCRIPT_SRC);
```

Comment:
This won't work on Mobile Wiki. Take a look at:
https://github.com/Wikia/mobile-wiki/blob/dev/front/main/app/components/article-content.js#L302-L361

You can return a div with `data-wikia-widget=playbuzz` and `data-item=123` and Ember will:
1. create `components/widget-playbuzz`
1. pass `item` attribute
1. replace your div with the rendered component